src/agentic/actor_agents.py

```diff
@@ -681,6 +681,60 @@ def get_name(f):
     def handle_request(self, method: str, data: dict):
         return f"Actor {self.name} processed {method} request with data: {data}"
 
+    async def webhook(self, run_id: str, callback_name: str, args: dict) -> Any:
```

Comment:
Note that all the other API methods are synchronous, and the calling code is also synchronous. We will need to do a "convert to async" pass on the code at some point, but I would stick with sync for now. Because we use _Ray_, it uses processes for concurrency, and so async is less important.